components/infobox/wikis/overwatch/infobox_league_custom.lua

```diff
@@ -61,16 +61,8 @@ function CustomInjector:parse(id, widgets)
 	)
 	elseif id == 'customcontent' then
 		if String.isNotEmpty(args.map1) then
-			local game = String.isNotEmpty(args.game) and ('/' .. args.game) or ''
-			local maps = {}
-
-			for _, map in ipairs(League:getAllArgsForBase(args, 'map')) do
-				table.insert(maps, tostring(self.caller:_createNoWrappingSpan(
-					PageLink.makeInternalLink({}, map, map .. game)
-				)))
-			end
 			table.insert(widgets, Title{name = 'Maps'})
-			table.insert(widgets, Center{content = {table.concat(maps, '&nbsp;• ')}})
+			table.insert(widgets, Center{content = self.caller:_makeBasedListFromArgs('map')})
```

Comment:
```suggestion
			local maps = Array.map(self.caller:getAllArgsForBase(args, 'map'), function(map)
				return PageLink.makeInternalLink(map)
			end)
			Array.appendWith(widgets,
				Logic.isNotEmpty(maps) and table.insert(widgets, Title{name = 'Maps'}) or nil,
				Center{content = table.concat(maps, '&nbsp;• ')}
			)
```

i suggest to only display the maps header if there are maps
additionally the below function seems quite redundant as a getAllArgsForBase together array.map and table.concat does the same and is cleaner and shorter code style wise